website/content/docs/job-specification/action.mdx

```diff
@@ -0,0 +1,127 @@
+---
+layout: docs
+page_title: action Block - Job Specification
+description: The "action" block allows for executable commands that operators
+
+---
+
+# `action` Block
+
+<Placement groups={['job', 'group', 'task', 'action']} />
+
+The `action` block allows job authors to define custom commands. These commands
+can be executed by operators with the necessary permissions on a running
+allocation, offering a controlled way to interact with tasks.
+
+## `action` Parameters
+
+The `action` block is used within a task and has the following parameters:
+
+- `command` (Required): Specifies the command to be executed.
+- `args` (Optional): Provides a list of arguments to pass to the command.
+
+```hcl
+job "my-job" {
+  group "my-group" {
+    task "my-task" {
+      action "get-changelog" {
+        command = "/usr/bin/curl"
+        args = ["-s", "https://raw.githubusercontent.com/hashicorp/nomad/main/CHANGELOG.md"]
+      }
+    }
+  }
+}
+```
+
+## `action` Examples
+
+### Basic Action
+This example demonstrates a simple action that prints the current date and time:
+
+```hcl
+job "example" {
+  ...
+  group "demo" {
+    ...
+    task "show-date" {
+      ...
+      action "current-date" {
+        command = "/bin/date"
+      }
+    }
+  }
+}
+```
+
+### Action with Arguments
+Here, an action uses arguments to perform a specific task:
+
+```hcl
+job "example" {
+  ...
+  group "demo" {
+    ...
+    task "list-files" {
+      ...
+      action "list-tmp" {
+        command = "/bin/ls"
+        args    = ["-l", "/tmp"]
+      }
+    }
+  }
+}
```

Comment:
```suggestion
job "example" {
  # ...
  group "demo" {
    # ...
    task "list-files" {
      # ...
      action "list-tmp" {
        command = "/bin/ls"
        args    = ["-l", "/tmp"]
      }
    }
  }
}
```

Same as above but GitHub was not happy to have codeblocks inside the suggestion 😅